lib/classes/Service.js

```diff
@@ -65,6 +76,14 @@ class Service {
       .parse(serviceFilePath)
       .then((serverlessFileParam) => {
         const serverlessFile = serverlessFileParam;
+
+        // check for non recognized attributes
+        Object.keys(serverlessFile).forEach((key) => {
+          if (this.allowedProperties.indexOf(key) === -1) {
+            throw new ServerlessError(`"${key}" is not recognized. If you are trying to use a custom attribute please use the "custom" object.`); // eslint-disable-line max-len
```

Comment:
One minor thing: I'd split that into multiple lines just like here: https://github.com/serverless/serverless/pull/3959/files#diff-1a6cb15a7aef359bf1fabed7982e4af3R107-L111